packages/forms-web-app/src/routes/config.js

```diff
@@ -255,8 +255,19 @@ const routesConfig = {
 				id: 'examination-select-file',
 				name: 'Select a file',
 				route: '/select-a-file',
+				view: 'pages/examination/select-file',
 				sessionId: 'files',
-				view: 'pages/examination/select-file'
+				onError: {
+					message: {
+						isEmpty: (fileToUploadName = '') => `${fileToUploadName} is empty`,
+						noFileSelected: 'Select a File',
+						checkMimeType: (fileToUploadName = '') =>
+							`${fileToUploadName} must be a JPG, BMP, PNG, TIF, TIFF, DOC, JPEG, XLS, XSLX or PDF.`,
```

Comment:
Think DOCX needs to be added.

Reply:
the api doesn't accept `bmp`, can add if needed though but wasn't on the list i was referring to (the one on the jira ticket - https://pins-ds.atlassian.net/browse/ASB-583)